src/components/screens/DraggableGrid.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactElement, useContext } from 'react';
+import Animated, {
+  useAnimatedRef,
+  useAnimatedScrollHandler,
+  useSharedValue
+} from 'react-native-reanimated';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { device, normalize } from '../../config';
+import { OrientationContext } from '../../OrientationProvider';
+
+import { DraggableItem, Positions } from './DraggableItem';
+
+type Props = {
+  children: ReactElement<{ draggableId: string; key: string }>[];
+  onDragEnd: (diff: Positions) => void;
+};
+
+export const DraggableGrid = ({ children, onDragEnd }: Props) => {
+  const scrollY = useSharedValue(0);
+  const scrollView = useAnimatedRef<Animated.ScrollView>();
+  const positions = useSharedValue<Positions>(
+    Object.assign(
+      {},
+      ...children.map((child, index) => ({ [child.props.draggableId?.replace('​', '')]: index }))
+    )
+  );
```

Comment:
do you mean replace with place?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace?retiredLocale=de
in our case we need to do that because of the fact i described in the commit:

> - added replacement for invisible delimiter with `.replace('​', '')`,
  that can be added per static contents to achieve multiline titles,
  to properly match strings in objects for sorting with `draggableId`

to make it more clear, here a screenshot of the data:

<img width="375" alt="image" src="https://user-images.githubusercontent.com/1942953/203022751-31dc9067-7dec-457c-b2c2-46bc0db03cf9.png">

Reply:
Thank you for your answer. 👍